ckanext/datastore/backend/postgres.py

```diff
@@ -1621,6 +1642,11 @@ def delete_data(context: Context, data_dict: dict[str, Any]):
     validate(context, data_dict)
     fields_types = _get_fields_types(
         context['connection'], data_dict['resource_id'])
+    fields = _get_fields(context['connection'], data_dict['resource_id'])
+    field_names = _pluck('id', fields)
+    sql_columns = ", ".join(
+        identifier(name) for name in field_names)
```

Comment:
`_pluck` shouldn't exist, please use something like this instead.

```suggestion
    sql_columns = ", ".join(
        identifier(f['id']) for f in fields)
```